workspace.code-workspace

```diff
@@ -0,0 +1,25 @@
+{
+  "folders": [
+    {
+      "path": "backend"
+    },
+    {
+      "path": "frontend"
+    },
+    {
+      "path": ".",
+      "name": "root"
+    }
+  ],
+  "settins": {
+    "[elm]": {
+      "editor.formatOnSave": true
+    },
+    "[haskell]": {
+      "editor.formatOnSave": true
+    }
+  },
+  "extensions": {
+    "recommendations": ["haskell.haskell", "elmtooling.elm-ls-vscode"]
```

Comment:
Can you add ormolu here too?

Reply:
haskell.haskell already supports formatting via ormolu. Why should another extension be added?

Reply:
`sjurmillidahl.ormolu-vscode`